Library/Formula/ems-flasher.rb

```diff
@@ -0,0 +1,19 @@
+class EmsFlasher < Formula
+  desc "Software for flashing the EMS Gameboy USB cart"
+  homepage "https://lacklustre.net/projects/ems-flasher/"
+  url "https://lacklustre.net/projects/ems-flasher/ems-flasher-0.03.tgz"
+  sha256 "d77723a3956e00a9b8af9a3545ed2c55cd2653d65137e91b38523f7805316786"
+  head "git://lacklustre.net/ems-flasher"
+
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+
+  def install
+    system "make"
+    bin.install "ems-flasher"
```

Comment:
There is no `configure` included with the software, and the stable version doesn't even have a `make install` target or manfile (but the HEAD version has both). Does this mean it still would be *better* for me to write separate patches for both of them, and use the patched `make install` targets, instead of using `bin.install` and `man1.install`?